projects/core/src/occ/adapters/cms/occ-cms-component.adapter.spec.ts

```diff
@@ -207,6 +207,58 @@ describe('OccCmsComponentAdapter', () => {
       testRequestPOST.flush(componentList);
     });
 
+    it('should use a post request when get fails to get a list of cms component data without pagination parameters - 405', () => {
```

Comment:
Let's refactor this test to reuse the code in the previous test instead of duplicating the test. You have two tests that look exactly the same, the only thing that changes is the http status